content/learn/quick-start/getting-started/setup.md

```diff
@@ -246,6 +246,33 @@ channel = "nightly"
 
 For more information, see [The rustup book: Overrides](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file).
 
+#### Cranelift
+
+This uses a new nightly-only codegen that is about 30% faster at compiling than LLM. 
+
+To install cranelift on Linux or macOS, run the following.
+```
+rustup component add rustc-codegen-cranelift-preview --toolchain nightly
+```
+
+To activate it for your project, add the following to your `.config/cargo.toml`.
+```toml
+[unstable]
+codegen-backend = true
+
+# Replace `my_program` with the name of your binary.
+[profile.dev.package.my_program]
+codegen-backend = "cranelift"
+```
+
+This enables faster compiles for your binary, but builds Bevy and other dependencies with the more-optimized LLVM backend. See the [cranelift setup guide](https://github.com/rust-lang/rustc_codegen_cranelift?tab=readme-ov-file#download-using-rustup) for
+details on other ways in which cranelift can be enabled. The installation process for Windows is a bit more involved. Consult the linked documentation for help.
+Note that cranelift [currently does not build Bevy on macOS](https://github.com/rust-lang/rustc_codegen_cranelift/issues/1504).
```

Comment:
I don't have time to work on this today, but this bug shouldn't be too hard to fix. I did prefer it gets fixed before merging this PR.